### PR TITLE
Add dependency status to README

### PR DESCRIPTION
**Pull request by iankronquist**
I was going to include this with my other commits but I guess they got merged already. If you think this is ugly or don't trust the service, fine, I don't care.

### CODE REVIEW

**Comment by tschuy** [comment]
Is there any way to get them to display on the same line?

**Comment by iankronquist** [comment]
We could do this: https://github.com/dotnet/roslyn
Or this: https://raw.githubusercontent.com/badges/shields/master/README.md

Readmes are supposed to be human readable in plaintext. I vote it stays as is.

**Comment by tschuy** [comment]
What about just putting the image elements side by side? Like 

```
 ![travis](https://travis-ci.org/osuosl/timesync.svg?branch=develop) [![Dependency Status](https://david-dm.org/osuosl/timesync.svg)](https://david-dm.org/osuosl/timesync)
```

**Comment by iankronquist** [comment]
Oh, apparently that works. Also, grip is pretty cool.
https://pypi.python.org/pypi/grip

**Comment by ElijahCaine** [comment]
@iankronquist I ran the line ``npm run fixtures`` and got the following output. 
```
> timesync@0.0.0 fixtures /home/voigte/gitsrc/other/timesync
> node ./scripts/load_fixtures.js

Unhandled rejection Error: SQLITE_ERROR: no such table: activities
    at Error (native)
```
Just wanted to let you know. If we just need to run a few commands before that to guarantee it works let's add that to the README.

**Comment by tschuy** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -26,8 +26,38 @@ $ npm test
 $ npm run linter
 ```
 
+To make a quick request on the dev instance, first load the fixtures:
+```
+$ npm run fixtures
```
Comment (ElijahCaine) [inline README.md]: When I run the line ``npm run fixtures`` I get the following output: 
```
$ npm run fixtures

> timesync@0.0.0 fixtures /home/voigte/gitsrc/other/timesync
> node ./scripts/load_fixtures.js

Unhandled rejection Error: SQLITE_ERROR: no such table: activities
    at Error (native)
```

when I run the later line ``curl -XGET -s localhost:8000/v1/times | python -m json.tool`` (in a different terminal) I get this output:
```
$ curl -XGET -s localhost:8000/v1/times | python -m json.tool
No JSON object could be decoded 
```
I'm sure there is a little prep work to make this command work, we should add that to this PR.

If there is not prep work and this is just *theoretical* functionality we should note that for the reader.

Comment (iankronquist) [inline README.md]: It looks like I forgot to mention migrations. Thanks for pointing this out.

**Comment by iankronquist** [comment]
@ElijahCaine revised according to your comments. r?

File: README.md [hunk 2]
```diff
@@ -26,8 +26,47 @@ $ npm test
...
+and load the fixtures:
+```
+$ npm run migrations
+$ npm run fixtures
+```
+
+Next, run the application:
+
+```
+$ curl -XGET -s localhost:8000/v1/times | python -m json.tool
```
Comment (tschuy) [inline README.md]: So this is how I run the application?

Comment (ElijahCaine) [inline README.md]: @iankronquist If you add a quick line in between the run fixtures line and the curl line that says "in a seperate terminal start timesync with the ``npm start`` command" I'll give it my +1.

Comment (iankronquist) [inline README.md]: I think I got distracted. Fixed.

**Pull request merged**